vitrina/settings.py

```diff
@@ -28,7 +28,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['*']
```

Comment:
```suggestion
ALLOWED_HOSTS = ['data.gov.lt', 'staging.data.gov.lt']
```

Please provide explicit list of allowed hosts to prevent host header attacks.

https://docs.djangoproject.com/en/3.2/ref/settings/#allowed-hosts

Also you can add `localhost` if that was the reason for adding `'*'`.